src/base/components/FeedList.js

```diff
@@ -25,8 +32,83 @@ class FeedListPage extends React.Component {
         tspan.setAttribute('x', 90);
       }
     }, 3000)
-  }
 
+    let connection = new signalR.HubConnectionBuilder()
+    .configureLogging(signalR.LogLevel.Debug)
+  .withUrl(gconfig.gatewaysServiceUrl+'/devHub')
+  .build();
+      
+    connection.on("ReceiveMessage", (topic ,jdata) => {
+          console.log("ReceiveMessage:["+topic+"] "+ jdata);
+          
+          if(topic === "/iot/zolertia/replay"){
+            jdata =  NormailizeJSON(jdata);
+            var data = JSON.parse(jdata);
+            if(data.Command == "on" && data.Result == "Success"){
+              toast.success("Zolertia Successfully Turned ON!!!");
+            }
+            else if(data.Command == "off" && data.Result == "Success"){
+              toast.success("Zolertia Successfully Turned OFF!!!");
+            }
+            else if(data.Command == "SetTX" && data.Result == "Success"){
+              toast.success("Successfully Set "+data.Value+" to TX Level!!!");
+            }
+            else if (data.Command == "SetCh" && data.Result ==  "Success"){
+              toast.success("Successfully Set "+data.Value+" Channel!!!");
+            }
+            else if(data.Commnad == "SetTX" && data.Result == "Failed"){
+              toast.error("Failed to Set "+data.Value+" to TX Level!!!");
+            }
+            else if (data.Command == "SetCh" && data.Result ==  "Failed"){
+              toast.error("Failed to Set "+data.Value+" Channel!!!");
+            }
+          }
+          else if(topic === "/iot/sensor/data" ){
+            jdata =  NormailizeJSON(jdata);
+            var data = JSON.parse(jdata);
+            this.setState({ temperature: (parseFloat(data.Temperature)/1000)});
+            this.setState({ battery: (parseFloat(data.BatteryVal)/1000)});
+            this.setState({ devicestatus: "off"});
+          }
+          else if(topic === "/iot/sensor/boot" && jdata.includes("BOOTUP")){
+            toast.success("Zolertia Sucessfully Bootup now!!!");
+          }
+          else if(topic === "/iot/sensor/error"){
+            if(jdata.includes("SERIALACCESSERROR")){
+              toast.error("Zolertia does not FOUND!!!");
+              this.setState({ devicestatus: "on"});
+            }            
+            if(jdata.includes("SERIALREADWRITEERROR")){
+              toast.error("Zolertia read write error FOUND!!!");
```

Comment:
Not sure what does this message mean, please clerify

Reply:
During read/write from Device of if any error comes up, then trigger this event, its very rare